tensorflow_data_validation/api/stats_api_test.py

```diff
@@ -43,6 +44,7 @@ class StatsAPITest(absltest.TestCase):
   def _get_temp_dir(self):
     return tempfile.mkdtemp()
 
+  @pytest.mark.xfail(run=True, reason="PR 260 This test fails and needs to be fixed.")
```

Comment:
In other repos, I opted not to run the xfailed tests because they took a long time. If that's not an issue here, then leave the marks as they are. Otherwise, you might consider not running them. This applies to all the xfail marks.